agent/config/config_test.go

```diff
@@ -155,7 +155,16 @@ func TestConfigBoolean(t *testing.T) {
 }
 
 func TestConfigDefault(t *testing.T) {
-	cfg := DefaultConfig()
+	os.Unsetenv("ECS_DISABLE_METRICS")
```

Comment:
I wanted to be more explicit about which variables were getting unset so that it's immediately obvious when reading the code.